pkg/executor/executortype/poolmgr/gp.go

```diff
@@ -538,7 +548,10 @@ func (gp *GenericPool) getFuncSvc(ctx context.Context, fn *fv1.Function) (*fscac
 		// Remove old versions function pods
 		for _, pod := range podList.Items {
 			// Delete pod no matter what status it is
-			gp.kubernetesClient.CoreV1().Pods(gp.namespace).Delete(pod.ObjectMeta.Name, nil)
+			err := gp.kubernetesClient.CoreV1().Pods(gp.namespace).Delete(pod.ObjectMeta.Name, nil)
```

Comment:
I think this has an impact on functionality, not sure if we should return if one of the pods in list has errors. Can we skip this change?

Reply:
Done.